share/poly_test.go

```diff
@@ -317,3 +318,132 @@ func TestPriPolyCoefficients(test *testing.T) {
 	require.Equal(test, a.coeffs, b.coeffs)
 
 }
+
+func TestRefreshDKG(test *testing.T) {
+	g := edwards25519.NewBlakeSHA256Ed25519()
+	n := 10
+	t := n/2 + 1
```

Comment:
Would it be interesting/useful to test with other values for t? Where is the code that says, "error t is too small, can't do what you are asking for"?

Reply:
Actually now that you mention it: The refreshing phase should be done with new values for `t` and `n` as this might happen in real use cases.